common/types/message/message.go

```diff
@@ -39,20 +42,26 @@ const (
 	ProofTypeBundle
 )
 
-// ChunkTaskDetail is a type containing ChunkTask detail.
-type ChunkTaskDetail struct {
+// LegacyChunkTaskDetail is a type containing ChunkTask detail.
+type LegacyChunkTaskDetail struct {
 	BlockHashes []common.Hash `json:"block_hashes"`
 }
 
+// EuclidV2ChunkTaskDetail is a type containing ChunkTask detail for euclid v2 chunk task.
+type EuclidV2ChunkTaskDetail struct {
+	BlockHashes      []common.Hash `json:"block_hashes"`
+	PrevMsgQueueHash common.Hash   `json:"prev_msg_queue_hash"`
+}
+
 // BatchTaskDetail is a type containing BatchTask detail.
 type BatchTaskDetail struct {
-	ChunkInfos    []*ChunkInfo `json:"chunk_infos"`
-	ChunkProofs   []ChunkProof `json:"chunk_proofs"`
-	BatchHeader   interface{}  `json:"batch_header"`
-	BlobBytes     []byte       `json:"blob_bytes"`
-	KzgProof      []byte       `json:"kzg_proof"`
-	KzgCommitment []byte       `json:"kzg_commitment"`
-	Challenge     common.Hash  `json:"challenge"`
+	ChunkInfos      []*ChunkInfo `json:"chunk_infos"`
+	ChunkProofs     []ChunkProof `json:"chunk_proofs"`
+	BatchHeader     interface{}  `json:"batch_header"`
+	BlobBytes       []byte       `json:"blob_bytes"`
```

Comment:
Can blob bytes being marshalled into base64 (prover expect base64 encoding here)

Reply:
It's a legacy field and can pass the batch task proving, so no problem in this field?